pkg/mock/verifications_test.go

```diff
@@ -0,0 +1,60 @@
+package mock
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestVerifications(t *testing.T) {
```

Comment:
So what do we get back as an error message if this verification fails? We were getting 406 from the mock server directly before which was confusing.

Reply:
There is an assertion in the `doCall`. I'll have to test this on the real MockServer to be sure it works correctly.